qutebrowser/browser/hints.py

```diff
@@ -124,7 +124,10 @@ def _move_to_elem(self):
             return
         no_js = config.val.hints.find_implementation != 'javascript'
         rect = self.elem.rect_on_view(no_js=no_js)
-        self.move(rect.x(), rect.y())
+        if rect.y()<0 and (rect.y()+rect.height()-15>0):
```

Comment:
I don't think hard-coding 15 here is a good idea.

We can get the current height of the label with

`self.height()`

however, at this point, the label dosen't seem to have been scaled to it's contents yet, and seems to have the default size.

One option here is to call `self.adjustSize()` to size the element before calling the height function. Note that there will be no text at this point, so the hint's width will be incorrect, but I think the height should be right. When calling `adjustSize` before `height`, I get a value of 16 on my computer (which may vary depending on the fonts used, for example).